runtime/doc/lua.txt

```diff
@@ -830,7 +831,7 @@ vim.types						*vim.types*
 LUA-VIMSCRIPT BRIDGE                                    *lua-vimscript*
 
 Nvim Lua provides an interface to Vimscript variables and functions, and
-editor commands and options.
+editor commands and options. See also https://github.com/nanotee/nvim-lua-guide.
```

Comment:
Sorry for being nitpicky about this, but better put this on a newline, since we usually target max 78 cols (last line in doc usually does `vim: tw=78`). It doesn't always works out, especially not for autogenerated stuff, but here it does.

:)

Reply:
No worries; fixed. (It did break, but I had `<....>` tags before, and didn't check length again after removing them.)